crates/parcel/src/requests/asset_request.rs

```diff
@@ -141,7 +142,7 @@ fn run_pipeline(
     if is_different_asset_type {
       let next_pipeline = plugins.transformers(transform_input.file_path(), None)?;
 
-      if next_pipeline != pipeline {
+      if next_pipeline.hash() != pipeline_hash {
```

Comment:
Why did you make this change?

Reply:
Because the plug-in trait needs to take a mutable ref. (to enable mutability further one)

With partial-eq, we need to keep the entire `pipeline` as a read ref. so it can't be mutable as internally in the equality check it could have access to all the data within the `pipeline` object. 

If we externalise the check we're able to make this requirement looser.